nucypher/control/emitters.py

```diff
@@ -255,25 +263,35 @@ def exception(self,
                   log_level: str = 'info',
                   response_code: int = 500):
 
-        exception = f"{type(e).__name__}: {str(e)}" if str(e) else type(e).__name__
-        message = f"{self} [{str(response_code)} - {error_message}] | ERROR: {exception}"
-        logger = getattr(self.log, log_level)
-        # See #724 / 2156
-        message_cleaned_for_logger = message.replace("{", "<^<").replace("}", ">^>")
-        logger(message_cleaned_for_logger)
+        self._log_exception(e, error_message, log_level, response_code)
         if self.crash_on_error:
             raise e
 
         response_message = str(e) or type(e).__name__
         return self.sink(response_message, status=response_code)
 
-    def respond(self, response) -> Response:
-        assembled_response = self.assemble_response(response=response)
+    def exception_with_response(self,
+                                json_error_response,
+                                e,
+                                error_message: str,
+                                log_level: str = 'info',
+                                response_code: int = 500):
```

Comment:
Just to recap -- under what conditions do you expect a 500 to be raised?  I think it's important we honor the 500 series errors a true "server exceptions".

Reply:
✔️  - Generally, for the WebController any non-specific exceptions results in the response code of 500. In the future I think we should allow subclasses of WebController to specific an exception, http_code mapping so that it is more customizable - but that's later on.

For the only use of this method  the response code is specified (`404`) so 500 doesn't get used here. In any case, I removed the default value of `response_code` and it must now be explicitly specified when calling this function.